deployment/cfn/data_plane.py

```diff
@@ -512,14 +512,6 @@ def create_dns_records(self, bastion_host, rds_database,
             'dnsPrivateRecords',
             HostedZoneId=Ref(self.private_hosted_zone_id),
             RecordSets=[
-                r53.RecordSet(
```

Comment:
Good job recognizing that this was specific to internal to the VPC metrics and log aggregation.